assets/src/scripts/index.js

```diff
@@ -1,19 +1,24 @@
 // Initialize the 18F Web design standards
 require('uswds');
+const { timeFormat } = require('d3-time-format');
 
 const { getTimeseries } = require('./models');
 const Hydrograph = require('./hydrograph');
 
+// Create a time formatting function from D3's timeFormat
+const formatTime = timeFormat('%c %Z');
 
 function main() {
     let nodes = document.getElementsByClassName('hydrograph');
     for (let node of nodes) {
         getTimeseries({sites: [node.dataset.siteno]}, series => {
-            let dataIsValid = series[0] && !series[0].values.some(d => d.value == -999999);
+            let dataIsValid = series[0] && !series[0].values.some(d => d.value === -999999);
             new Hydrograph({
                 element: node,
                 data: dataIsValid ? series[0].values : [],
-                title: dataIsValid ? series[0].description : 'No data'
+                yLabel: dataIsValid ? series[0].description : 'No data',
```

Comment:
Somehow, running locally - the y axis label is now the default "Data" rather than the timeseries description.

Reply:
Thanks, I see that now. I'll fix